presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -370,6 +374,7 @@ private void invokeNoMoreSplitsIfNecessary()
 
         // S3 Select pushdown works at the granularity of individual S3 objects,
         // therefore we must not split files when it is enabled.
+        Properties schema = getHiveSchema(storage.getSerdeParameters(), table.getParameters());
```

Comment:
Based on the existing `getHiveSchema` semantics it can also be in the table parameters. I understand that this is rather unlikely, but i would like to keep the semantics the same in this PR.